packages/components/src/molecules/Panels/args.tsx

```diff
@@ -24,6 +24,20 @@ export const AllRowsArgs: ResizableRowsPanelType = {
   },
 };
 
+export const ClosableLeftColumnArgs: ResizableColumnsPanelType = {
+  left: <div style={{ background: "#002B7F", height: "100%", color: "#fff", padding: 15 }}>Left Panel</div>,
+  center: <div style={{ background: "#FCD116", height: "100%", padding: 15 }}>Center Panel</div>,
+  right: <div style={{ background: "#CE1126", height: "100%", padding: 15 }}>Right Panel</div>,
```

Comment:
I have a specific question @topliceanurazvan : Why do we set Height as 100% for all Div elements. I know it is better to set 100% for Body and Container elements in some specific cases/layouts, but as I see, we set 100% to many divs in our styles so far, question is: what is the purpose we do it for left-center-right panels if we can just set 100% for their parent container and they will inherit it?

Reply:
In this case, we have the 100% height for the container of the left pane, but if we don't set the height of the actual left pane to 100%, it will not inherit the full height but will look like this:

![image](https://user-images.githubusercontent.com/47887589/200525735-5d3c0362-0d16-4ef8-bf8d-82e1fdc20021.png)

The height will be given only by the padding and the font-size of the text ( 52 px in the screenshot, instead of 100% )

Reply:
Oh, alright, I see. 
Later I may refactor html/css side and normalize its behavior.
Btw, have you tried setting` height: inherit `on left panel?

Reply:
Yes, it will give the same result as `100%` 😄 . Since this is used just for the storybook component, it shouldn't matter that much